doc/guides/dts/macros.bnf

```diff
@@ -39,16 +39,20 @@ node-macro =/ %s"DT_N" path-id %s"_IRQ_IDX_" DIGIT
               %s"_VAL_" dt-name [ %s"_EXISTS" ]
 node-macro =/ %s"DT_N" path-id %s"_IRQ_NAME_" dt-name
               %s"_VAL_" dt-name [ %s"_EXISTS" ]
-; For fixed-partitions give a unique ordinal value for each partition */
+; Subnodes of the fixed-partitions compatible get macros which contain
+; a unique ordinal value for each partition
 node-macro =/ %s"DT_N" path-id %s"_PARTITION_ID" DIGIT
-; Macros are generated for each of the node's compatibles.
+; Macros are generated for each of a node's compatibles;
+; dt-name in this case is something like "vnd_device".
 node-macro =/ %s"DT_N" path-id %s"_COMPAT_MATCHES_" dt-name
-; The node identifier for the node's parent in the devicetree.
+; Every node gets one of these macros which expands to the node identifier
```

Comment:
except the root node, if we want to get picky.  Maybe it's not a node.

Reply:
It's a node and this is an ABNF, so we should definitely be picky :)